devel/universal-ctags/Portfile

```
@@ -3,10 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        universal-ctags ctags feffe43a4a9d339cd6701744589746e2297e4567
+github.setup        universal-ctags ctags 0837c42c3fea8854499cf1d22f2f5de12d745f20
```

Since universal-ctags project provides git tags for releases, I suggest using it instead of commit hash. Also I think github.setup's tag_prefix (p in this case) is well suited here.
```suggestion
github.setup        universal-ctags ctags 5.9.0-20201213.0 p
```